queries.sql

```diff
@@ -1,8 +1,68 @@
+/*Queries that provide answers to the questions from all projects.*/
+
 SELECT * FROM animals WHERE name LIKE '%mon';
 SELECT * FROM animals WHERE neutered = true;
 SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
 SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
-SELECT * FROM animals WHERE name != 'Gabumon';
\ No newline at end of file
+SELECT * FROM animals WHERE name != 'Gabumon';
+
+
+BEGIN;
+update animals set species = 'unspecified';
+select * from animals;
+rollback;
+select * from animals;
+
+UPDATE animals set species = 'digimon' where name like '%mon';
+select * from animals;
+update animals set species = 'pokemon' where species = '';
+select * from animals;
+
+BEGIN;
+delete from animals;
+ROLLBACK;
+select * from animals;
+=======
+SELECT name FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+/* Query to update and read the animals table data */
+BEGIN;
+SELECT species FROM animals; -- verify original values
+UPDATE animals SET species = 'unspecified';
+SELECT species FROM animals; -- verify changes made
+ROLLBACK;
+SELECT species FROM animals; -- verify that change was rolled back
+COMMIT;
+
+BEGIN;
+UPDATE animals
```

Comment:
* According to the [requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md#:~:text=Write%20queries%20(using,queries.sql%20file) you are required to write queries that answer these below 👇 Kindly add them 🙏 
* Write queries (using JOIN) to answer the following questions:

> What animals belong to Melody Pond?
> List of all animals that are pokemon (their type is Pokemon).
> List all owners and their animals, remember to include those that don't own any animal.
> How many animals are there per species?
> List all Digimon owned by Jennifer Orwell.
> List all animals owned by Dean Winchester that haven't tried to escape.
> Who owns the most animals?
> Remember all these should be written in queries.sql file.